_spinnaker/pacrd.md

```diff
@@ -0,0 +1,372 @@
+"Bake Manifest" stage you would structure your definition like so:
+  
+```yaml
+# ...
+stages:
+  - type: BakeManifest
+    bakeManifest:
+      name: Bake the Bread
+      # ...
+# ...
+```
+
+- Documentation for available Pipeline spec fields must unfortunately be
+found in the installation manifest for this controller. You can do so by
+grepping for `pipelines.pacrd.armory.spinnaker.io` in the installation
+manifest. Fields are documented under `spec.validation.openAPIV3Schema`.
+
+### Creating
+
+Pipelines in Kubernetes can be defined with the same options you would define
+in the UI. The following example shows how you might define a trivial Pipeline:
+
+```yaml
+# file: pipeline.yaml
+apiVersion: pacrd.armory.spinnaker.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: myapplicationpipeline
+spec:
+  description: Delivery pipeline for the MyApplicationName service.
+  application: myapplicationname
+  stages:
+    # Note: In `v0.1.x` you are required to sepcify _both_ `type: BakeManifest`
+    #       as well as place options under a `bakeManifest` key. Consult the
+    #       "Current Limitations" section above for more information.
+    - type: BakeManifest
+      bakeManifest:
+        name: Bake Application
+        evaluateOverrideExpressions: false
+        outputName: myapplicationname
+        templateRenderer: helm2
+        refId: "1"
+    - type: ManualJudgment
+      manualJudgment:
+        name: Bake Successful?
+        comments: Was the bake successful?
+        failPipeline: true
+        instructions: Check to see if the helm template was baked correctly
+        refId: "2"
+        requisiteStageRefIds: [ "1" ]
+```
+
+_Note: this example assumes that you've created a `myapplicationname` application
+from the [previous section](#applications). If you haven't already, create one
+quickly before proceeding._
+
+If you've defined this in a `pipeline.yaml` file then you can create it in your
+cluster by running the following `kubectl` command:
+
+```
+kubectl apply -f pipeline.yaml
+```
+
+Once applied, you can check on the status of your pipeline by using either the
+`get` or `describe` commands. `kubectl` will recognize either `pipe` or
+`pipeline` for the resource kind:
+
+```sh
+❯ kubectl get pipe myapplicationpipeline
+NAME                    STATUS    LASTCONFIGURED   URL
+myapplicationpipeline   Updated   5s               http://localhost:9000/#/applications/myapplicationname/executions/configure/f1eb82ce-5a8f-4b7a-9976-38e4aa022702
+
+# or...
+
+
+❯ kubectl get pipeline myapplicationpipeline
+NAME                    STATUS    LASTCONFIGURED   URL
+myapplicationpipeline   Updated   44s              http://localhost:9000/#/applications/myapplicationname/executions/configure/f1eb82ce-5a8f-4b7a-9976-38e4aa022702
+```
+
+A `describe` call will give you additional contextual information about the
+status of your pipeline:
+
+```
+❯ kubectl describe pipeline myapplicationpipeline
```

Comment:
I generally advise people to separate inputs and outputs for the copy/pasters out there,e.g.:

Run the following command:
xyz

The command returns information similar to the following:
abc

Is there a reason we should keep them combined in this doc?

Reply:
Other than inertia, not much. We can give it a shot